src/meta/processors/job/JobManager.cpp

```diff
@@ -643,6 +643,10 @@ ErrorOr<nebula::cpp2::ErrorCode, std::vector<cpp2::JobDesc>> JobManager::showJob
         continue;
       }
       auto jobDesc = optJob.toJobDesc();
+      if (jobDesc.get_type() == cpp2::JobType::ZONE_BALANCE) {
```

Comment:
this way is also correct, but I prefer just modify this place SubmitJobExecutor.cpp:82,  what do you think

Reply:
Emm, it's better to not change jobdesc but only show name. Thanks.